lib/rocket_chat/messages/channel.rb

```diff
@@ -68,7 +68,7 @@ def online(room_id: nil, name: nil)
       # * [String] topic A room's topic
       # * [Strong] type c (channel) or p (private group)
       def self.settable_attributes
-        %i[description join_code purpose read_only topic type]
+        %i[custom_fields description join_code purpose read_only topic type]
```

Comment:
It would also be nice to see the other "new" setter attributes included. ie

```suggestion
        %i[announcement custom_fields default description join_code purpose read_only topic type]
```

Reply:
And could you please also update the documentation in the comment above? likely:
```
* [String] announcement Announcement for the channel
* [Hash] custom_fields Custom fields for the channel
* [Boolean] default Sets whether the channel is a default channel or not
```

Reply:
Oh, sure!